angular/projects/admin-nrpti/src/app/shared/shared.module.ts

```diff
@@ -3,17 +3,17 @@ import { BrowserModule } from '@angular/platform-browser';
 import { MatSnackBarModule, MatSlideToggleModule } from '@angular/material';
 
 import { DatePipe } from '@angular/common';
-import { OrderByPipe } from './pipes/order-by.pipe';
-import { NewlinesPipe } from './pipes/newlines.pipe';
-import { ObjectFilterPipe } from './pipes/object-filter.pipe';
-import { LinkifyPipe } from './pipes/linkify.pipe';
+import { OrderByPipe } from '../pipes/order-by.pipe';
+import { NewlinesPipe } from '../pipes/newlines.pipe';
+import { ObjectFilterPipe } from '../pipes/object-filter.pipe';
+import { LinkifyPipe } from '../pipes/linkify.pipe';
 
-import { FileUploadComponent } from './file-upload/file-upload.component';
 
 @NgModule({
   imports: [BrowserModule, MatSlideToggleModule, MatSnackBarModule],
-  declarations: [OrderByPipe, NewlinesPipe, ObjectFilterPipe, LinkifyPipe, FileUploadComponent],
+  declarations: [OrderByPipe, NewlinesPipe, ObjectFilterPipe, LinkifyPipe],
   providers: [DatePipe],
-  exports: [MatSlideToggleModule, MatSnackBarModule, OrderByPipe, NewlinesPipe, LinkifyPipe, FileUploadComponent]
+  exports: [MatSlideToggleModule, MatSnackBarModule, OrderByPipe, NewlinesPipe, LinkifyPipe],
+  entryComponents: []
 })
-export class SharedModule {}
+export class SharedModule { }
```

Comment:
Not a priority issue: maybe just double check your vscode formatter settings, its still adding a space here when it shouldn't.  I thought we found and fixed that setting a while back, but maybe that was only for the function spacing.